po/cs.po

```diff
@@ -58,17 +58,70 @@ msgstr "Nastavit indikátory spuštěných aplikací"
 msgid "Customize opacity"
 msgstr "Přizpůsobení průhlednosti"
 
-#: appIcons.js:810
+#: appIcons.js:797
 msgid "All Windows"
 msgstr "Všechna okna"
 
+#: appIcons.js:808
+msgid "New Window"
+msgstr "Nové okno"
+
+#: appIcons.js:823
+msgid "Launch using Dedicated Graphics Card"
+msgstr "Spustit pomocí vyhrazené grafické karty"
+
+#: appIcons.js:851
+msgid "Remove from Favorites"
+msgstr "Odstranit z oblíbených"
+
+#: appIcons.js:857
+msgid "Add to Favorites"
+msgstr "Přidat mezi oblíbené"
+
+#: appIcons.js:868
+msgid "Show Details"
+msgstr "Zobrazit podrobnosti"
+
+#: appIcons.js:896 appIcons.js:914 Settings.ui.h:11
+msgid "Quit"
+msgstr "Ukončit"
```

Comment:
I have changed this to "Ukončit" (which is a correct translation that is already used in GNOME Shell) instead of "Zastavit".